app/frontend/src/views/Application.vue

```diff
@@ -1,12 +1,81 @@
 <template>
   <v-container>
-    <BaseSecure>My Applications > {{acronym}}</BaseSecure>
+    <BaseSecure>
+      <div class="app-breadcrumb">
+        <span>
+          <router-link :to="{ name: 'MyApps' }">My Applications</router-link>
+        </span>
+        <span>></span>
+        <span>{{acronym}}</span>
+      </div>
+      <v-tabs vertical class="mt-10 getok-tabs">
+        <v-tab>API Access</v-tab>
+        <v-tab>Team</v-tab>
+
+        <v-tab-item>
+          <v-card flat class="pl-12">
+            <h2>{{acronym}}</h2>
+            <ApiAccess />
+          </v-card>
+        </v-tab-item>
+
+        <v-tab-item>
+          <v-card flat class="pl-12">
+            <h2>{{acronym}}</h2>
+            <v-card-text>
+              <p class="mb-0">TBA. Users tab coming soon</p>
+            </v-card-text>
+          </v-card>
+        </v-tab-item>
+      </v-tabs>
+    </BaseSecure>
   </v-container>
 </template>
 
 <script>
+import ApiAccess from '@/components/apiAccess/ApiAccess.vue';
+
 export default {
   name: 'Application',
+  components: {
+    ApiAccess
+  },
   props: ['acronym']
 };
 </script>
+
+<style lang="scss">
+/* Breadcrumb */
+.app-breadcrumb {
+  margin-top: 0.5em;
+  span {
+    padding-right: 1em;
+  }
+}
+
+/* Tab (TODO: if this is used elsewhere in the program, make this style global) */
+.getok-tabs {
```

Comment:
Tabs are currently used in JWTDialog component as well. This may be worth putting into the main .scss.

Reply:
Do they do vertical or horizontal tabs? The changes so far for this one might have some specificity to vertical tabs with some padding I'll have to put in to match the mockups.
But yeah if we want this general then it should go into the main and then "getok-tabs" can be put on to tabs to opt in to the style (or something like that)
We can circle back comparing the 2 tab usages.

Reply:
The JWTDialog uses horizontal standard tabs. We can revisit this at a later time to try and standardize.